### PR TITLE
Updates a bunch of chemical descriptions because I was bored

### PR DESCRIPTION
**Pull request by Incoming5643**
Probably spelling errors. Will fix in the morning.

Also removes a double definition for /datum/reagent/fluorine

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1144,21 +1137,21 @@
 /datum/reagent/saltpetre
 	name = "Saltpetre"
 	id = "saltpetre"
-	description = "Volatile."
+	description = "Volatile. Controversial. Third Thing."
```
Comment (Jordie0608) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Is this intentional?

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1010,21 +1010,14 @@
 /datum/reagent/iodine
 	name = "Iodine"
 	id = "iodine"
-	description = "A slippery solution."
-	reagent_state = LIQUID
-	color = "#C8A5DC"
-
-/datum/reagent/fluorine
```
Comment (Jordie0608) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Quack

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: This space left intentionally blank

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -544,7 +544,7 @@
 /datum/reagent/fluorine
 	name = "Fluorine"
 	id = "fluorine"
-	description = "A highly-reactive chemical element."
+	description = "A comically-reactive chemical element. The univese does not want this stuff to exist in this form in the slightest."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >univese

Universe

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -556,21 +556,21 @@
...
-	description = "A chemical element."
+	description = "A soft silver metal that can easily be cut with a knife. It's not salt just yet so refrain from putting in on your chips."
 	reagent_state = SOLID
 	color = "#808080" // rgb: 128, 128, 128
 
 /datum/reagent/phosphorus
 	name = "Phosphorus"
 	id = "phosphorus"
-	description = "A chemical element."
+	description = "A ruddy red powder that burns readily. Though it comes in many colors, the general theme is always the same. "
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Extra space at the end here?

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 5]
```diff
@@ -556,21 +556,21 @@
 /datum/reagent/sodium
 	name = "Sodium"
 	id = "sodium"
-	description = "A chemical element."
+	description = "A soft silver metal that can easily be cut with a knife. It's not salt just yet so refrain from putting in on your chips."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Needs commas

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 6]
```diff
@@ -556,21 +556,21 @@
...
-	description = "A chemical element."
+	description = "A ruddy red powder that burns readily. Though it comes in many colors, the general theme is always the same. "
 	reagent_state = SOLID
 	color = "#832828" // rgb: 131, 40, 40
 
 /datum/reagent/lithium
 	name = "Lithium"
 	id = "lithium"
-	description = "A chemical element."
+	description = "A silver metal, it's claim to fame is its remarkabley low density. Using it is a bit too effective in calming oneself down."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >remarkabley
Remarkably

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 7]
```diff
@@ -1010,21 +1010,14 @@
...
 	id = "iodine"
-	description = "A slippery solution."
-	reagent_state = LIQUID
-	color = "#C8A5DC"
-
-/datum/reagent/fluorine
-	name = "Fluorine"
-	id = "fluorine"
-	description = "A slippery solution."
+	description = "Commonly added to table salt as a nutrient. On its own it tastes far less pleasing.."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Extra period

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 8]
```diff
@@ -1038,35 +1031,35 @@
 /datum/reagent/bromine
 	name = "Bromine"
 	id = "bromine"
-	description = "A slippery solution."
+	description = "A brownish liquid that's highly reactive. Useful for stopping free radicals, but not intend for human consumption."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >intend

Intended

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 9]
```diff
@@ -1038,35 +1031,35 @@
...
+	description = "An aromatic ring of carbon with a hydroxyl group. A useful precursor to some medicines, but has no healing properties on its own."
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 
 /datum/reagent/ash
 	name = "Ash"
 	id = "ash"
-	description = "Basic ingredient in a couple of recipes."
+	description = "Supposedly pheonixes rise from these, but you've never see it."
 	reagent_state = LIQUID
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >see

Seen

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 10]
```diff
@@ -1038,35 +1031,35 @@
...
-	description = "Common ingredient in other recipes."
+	description = "A slick, slightly carcinogenic liquid. Has a multitude of mundane uses in everyday life."
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 
 /datum/reagent/colorful_reagent
 	name = "Colorful Reagent"
 	id = "colorful_reagent"
-	description = "A solution."
+	description = "Throughly sample the rainbow."
```
Comment (PKPenguin321) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: >Throughly

I think you meant "Thoroughly?"

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (Incoming5643) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: univision

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 11]
```diff
@@ -865,7 +865,7 @@
 	name = "Carbon Dioxide"
 	id = "co2"
 	reagent_state = GAS
-	description = "A gas commonly produced by burning carbon fuels."
+	description = "A gas commonly produced by burning carbon fuels. You're constantly producing this in your lungs."
```
Comment (Incoming5643) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: Me resisting a meme.

**Comment by Incoming5643** [comment]
Man I guess I'm more tired than I thought.

Goon knight

:zzz:

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: stealth nerf, sound the alarms

Comment (Incoming5643) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: It'd double defined, this does literally nothing

**Pull request merged**